user_guide_src/source/tutorial/create_news_items.rst

```diff
@@ -90,7 +90,8 @@ Most helper functions require the helper to be loaded before use.
 Next, we check if we deal with the **POST** request with the
 :doc:`IncomingRequest <../incoming/incomingrequest>` object ``$this->request``.
 It is set in the controller by the framework.
-If the HTTP method is not POST, that is it is GET,
+The :ref:`IncomingRequest::is() <incomingrequest-is>` method checks the type of the request.
+If the request is not a POST request, that is it is a GET request,
```

Comment:
```suggestion
Since the route for **create()** endpoint handles both: **GET** and **POST** requests we can safely assume that if the request is not POST then it is a GET type.
```
Maybe we can add some context to this?

Reply:
Thank you! It is better.